src/utils/stringUtils.js

```diff
@@ -0,0 +1,9 @@
+const splitByComma = function splitByComma(inputString) {
+  return inputString.split(',');
+};
+
+const joinByComma = function joinByCommaFunc(inputString) {
+  return inputString.join(', ');
+};
```

Comment:
```suggestion
// 사용처에서 패턴을 주입하기 때문에 어떤 결과를 받을 수 있는지 유추할 수 있고 범용적으로 사용할 수 있음
const joinByPattern = function joinByCommaFunc(array, pattern) { 
  return array.join(pattern);
};
```
위에도 리뷰 남기긴했지만 저도 이렇게 추상화를 했던 적이 있었고 과한 추상화라는 결론을 하게 되었는데 만약에 이렇게 사용하더라도 `pattern`을 인자로 받아서 범용적으로 사용하는건 어떨까요?? 

`joinByComma`라는 함수 이름은 `쉼표(,)`를 기준으로 문자열로 합칠 것 같은데 내부적으로는 `, ` 이렇게 공백도 포함되어 있기 때문에 함수의 동작을 유추하기는 어려울 것 같다는 문제가 있을 것 같아요🤔

그리고 인자로 `inputString`을 받는데 string 타입을 다시 `join`하는게 살짝 어색하게 느껴지는 것 같아요

Reply:
확실히 패턴을 받아서 join을 하는 것이 더 좋은 코드 같아 보이네요 !! 좋은 의견 감사합니다 👍